### PR TITLE
improvement/gallery-item.sizing

### PR DESCRIPTION
**Pull request by Frankruiz90**
image
image
image

change the size of the image container so that it is zoomed in or out

### CODE REVIEW

**Comment by josemariogutierrez** [comment]
Please check zeplin, the spacing is wrong, text should be aligned to the title in the hero and have 20px of spacing to each side. In mobile gallery-items should have 20px between them, and 20px on each side.

File: css/gallery.css [hunk 1]
```diff
@@ -31,12 +37,12 @@
   background-position: center;
   background-size: cover;
   position: relative;
-  width: 227px;
-  height: 227px;
   margin-right: 17px;
   margin-bottom: 17px;
   box-shadow: 0px 0px 2px 0px rgba(0, 0, 0, 0.75);
   overflow: hidden;
+  flex-basis: 20%;
```
Comment (jusk02) [inline css/gallery.css]: its not for you to change it, but why 20%?

**Comment by jusk02** [comment]
rename this PR to the name of your branch please.

File: css/gallery.css (hunk 1 above)
Comment (Frankruiz90) [reply css/gallery.css]: so that the size of the container between the elements is resolved by 20% and there is 4 in line

**Comment by josemariogutierrez** [comment]
please make sure that in a minimum viewport size of 360px the number of columns are two, right now there is only one column.

File: css/gallery.css (hunk 1 above)
Comment (jusk02) [reply css/gallery.css]: but, 4 x 20% would be 80%, where would the other 20% be?

**Review by Sonny97 — APPROVED** [review]

File: css/gallery.css [hunk 2]
```diff
@@ -1,107 +1,130 @@
 /*class for the section of gallery*/
+.gallery-item::before {
+  content: " ";
+  display: block;
+  padding-top: 100%;
+}
 .gallery {
-  width: 635px;
-  height: 88px;
   font-family: Roboto;
```
Comment (jusk02) [inline css/gallery.css]: This line is not necesary, font is set in general in styles.css, i know you did not add it here but some cleanup comes well

File: css/gallery.css [hunk 3]
```diff
@@ -1,107 +1,130 @@
...
-}
-.gallery__text {
-  padding-bottom: 17px;
-  margin-block-start: 0px;
-  margin-block-end: 0px;
+  display: flex;
+  flex-flow: row wrap;
+  margin-left: 20px;
+  margin-right: 20px;
+  margin-bottom: 30px;
```
Comment (jusk02) [inline css/gallery.css]: you could replace ,bottom,left and right with `margin: '0 20px 30px`, this meaning top = 0, sides = 20 and bottom = 30

File: css/gallery.css [hunk 4]
```diff
@@ -1,107 +1,130 @@
...
+  padding-top: 100%;
+}
 .gallery {
-  width: 635px;
-  height: 88px;
   font-family: Roboto;
   font-size: 16px;
   color: black;
-  padding: 60px 0;
+  padding-right: 20px;
```
Comment (jusk02) [inline css/gallery.css]: besides the right padding you are missing 20px padding on the left, 30px padding at the top and 40px padding at the bottom

File: css/gallery.css [hunk 5]
```diff
@@ -1,107 +1,130 @@
...
-  cursor: pointer;
+.gallery__items :nth-child(2n) {
+  margin-right: 0;
 }
 /* start media queries */
-@media screen and (max-width: 960px) {
+@media screen and (min-width: 960px) {
+  .gallery {
+    width: 635px;
+    height: 88px;
```
Comment (jusk02) [inline css/gallery.css]: this height rule, does it serve some purpose, it seems it would cause problems

File: css/gallery.css [hunk 6]
```diff
@@ -1,107 +1,130 @@
...
+    padding: 60px 0;
```
Comment (jusk02) [inline css/gallery.css]: this padding only applies to top, its supposed to be 100px at the bottom not 60

File: css/gallery.css (hunk 5 above)
Comment (Frankruiz90) [reply css/gallery.css]: reviewing it does nothing

File: css/gallery.css (hunk 6 above)
Comment (Frankruiz90) [reply css/gallery.css]: image

Comment (jusk02) [reply css/gallery.css]: traduzco el anterior "este padding solo deberia aplicar al top, en la parte de abajo deberia ser de 100px, no 60"

Comment (jusk02) [reply css/gallery.css]: Screen Shot 2019-06-27 at 10 29 58 AM

File: css/gallery.css [hunk 7]
```diff
@@ -1,107 +1,150 @@
...
+  display: flex;
+  flex-flow: row wrap;
+  margin: 0 0 30px;
 }
+
 .gallery__items {
   display: flex;
   flex-flow: row wrap;
   justify-content: center;
+  min-width: 320px;
```
Comment (jusk02) [inline css/gallery.css]: this min width will cause overflow on small sizes, please remove it.

**Comment by jusk02** [comment]
image

Please check all of your margins and paddings.

**Review by josemariogutierrez — APPROVED** [review]

**Pull request merged**